src/components/Calculator.css

```diff
@@ -10,6 +10,21 @@
   width: 100%;
 }
 
+.container {
```

Comment:
- [ ] **Optional:** Although the aim of this project isn't to have an overly designed app, you should try to add more styling to make your work more visually appealing.

Reply:
Thank you @brytebee for the feedback. I'll make sure to work on that 👍